packages/ping-protect/src/lib/ping-protect.ts

```diff
@@ -0,0 +1,80 @@
+import './ping-signals-sdk.js';
+
+export interface Identifiers {
+  [key: string]: string;
+}
+
+/**
+ * Interface for the init method parameters
+ * All the parameters are optional
```

Comment:
The readme seems to suggest `envId` is required?
Not customer facing of course but want to check either way.